sdk/communication/azure-communication-phonenumbers/README.md

```diff
@@ -16,153 +16,129 @@ Install the Azure Communication Phone Numbers client library for Python with [pi
 
-All geographic phone plans within the same country are grouped into a phone plan group with a Geographic phone number type. All Toll-Free phone plans within the same country are grouped into a phone plan group.
+### Searching and Purchasing and Releasing numbers
 
-### Reserving and Acquiring numbers
+Phone numbers can be searched through the search creation API by providing an area code, quantity of phone numbers, application type, phone number type, and capabilities. The provided quantity of phone numbers will be reserved for ten minutes and can be purchased within this time. If the search is not purchased, the phone numbers will become available to others after ten minutes. If the search is purchased, then the phone numbers are acquired for the Azure resources.
 
-Phone numbers can be reserved through the begin_reserve_phone_numbers API by providing a phone plan id, an area code and quantity of phone numbers. The provided quantity of phone numbers will be reserved for ten minutes. This reservation of phone numbers can either be cancelled or purchased. If the reservation is cancelled, then the phone numbers will become available to others. If the reservation is purchased, then the phone numbers are acquired for the Azure resources.
+Phone numbers can also be released using the release API.
 
-### Configuring / Assigning numbers
+## Examples
 
-Phone numbers can be assigned to a callback URL via the configure number API. As part of the configuration, you will need an acquired phone number, callback URL and application id.
+### Get All Phone Numbers
 
-# Examples
-The following section provides several code snippets covering some of the most common Azure Communication Services tasks, including:
-
-## Communication Phone number
-### Get Countries
+Lists all of your acquired phone numbers
 
 ```python
-phone_number_administration_client = PhoneNumbersAdministrationClient.from_connection_string(connection_str)
-
-supported_countries = phone_number_administration_client.list_all_supported_countries()
-for supported_country in supported_countries:
-    print(supported_country)
+acquired_phone_numbers = phone_numbers_client.list_acquired_phone_numbers()
+acquired_phone_number = acquired_phone_numbers.next()
+print(acquired_phone_number.phone_number)
 ```
 
-### Get Phone Plan Groups
+### Get Phone Number
 
-Phone plan groups come in two types, Geographic and Toll-Free.
+Gets the information from the specified phone number
 
 ```python
-phone_number_administration_client = PhoneNumbersAdministrationClient.from_connection_string(connection_str)
-
-phone_plan_groups_response = phone_number_administration_client.list_phone_plan_groups(
-    country_code='<country code>'
-)
-for phone_plan_group in phone_plan_groups_response:
-    print(phone_plan_group)
+result = phone_numbers_client.get_phone_number("+18001234567")
```

Comment:
we had a bug filed in JS not to show numbers, even if they're obviously fake (unless the number itself is invalid format, is my interpretation)
https://github.com/Azure/azure-sdk-for-js/issues/12918